codesigning/enqueue-current-installer.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+#
+# Run this script to enqueue the windows binary for this current version of the
```

Comment:
Update comment to mention mac now too or leave ambiguous?

Reply:
Updated in 3dd189d56